test/FunctionalTests/Microsoft.OData.Core.Tests/ODataMessageWriterTests.cs

```diff
@@ -1102,6 +1176,79 @@ private static IEdmModel GetEdmModel()
             }
         }
 
+        /// <summary>
+        /// Write a message using an <see cref="ODataMessageWriter"/> instance
+        /// and returns the written payload as a string.
+        /// </summary>
+        /// <param name="edmModel">The <see cref="IEdmModel"/> used to initialize the writer.</param>
+        /// <param name="UseLegacyScaleAndSridVariable">Whether to use legacy scale and srid variable casing.</param>
+        /// <param name="test">The action that writes the payload.</param>

```

Comment:
I don't think this is a good choice of the parameter name given it's description

Reply:
I don't think the name holds any significance here. But it's a bool that if true the legacyscaleandsrid variables are used. Otherwise they are not.

Reply:
@ElizabethOkerio It does have significance otherwise there's no need to add a description. Parameter name is `test` and it's description is `The action that writes the payload`. I was not referring `UseLegacyScaleAndSridVariable` parameter if you look at the line where my comment.